recipes/enum-flags/0.1a/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, tools
+import os
+
+
+class EnumFlagsConan(ConanFile):
+    name = "enum-flags"
```

Comment:
```suggestion
    name = "enum-flags"
    version = "0.1a"
```
As you are using a specific folder, you can set a specific version here.

Reply:
Is it really required? I will have to remove it if a second version gets released (I did the same in all the PRs I've opened)

Reply:
I don't think it's required - we may add more versions, so let's keep as is